src/features/Auth/components/SignUp.jsx

```diff
@@ -0,0 +1,198 @@
+import {
+  Flex,
+  Heading,
+  Input,
+  Button,
+  InputGroup,
+  Stack,
+  InputLeftElement,
+  chakra,
+  Box,
+  Avatar,
+  FormControl,
+  InputRightElement,
+  useBoolean,
+  Text,
+  Spinner,
+  FormErrorMessage,
+} from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Link, useNavigate } from "react-router-dom";
+import { FaUserAlt, FaLock, FaUserCircle } from "react-icons/fa";
+import { MdEmail } from "react-icons/md";
+import { signupHandler } from "features";
+const CFaUserAlt = chakra(FaUserAlt);
+const CFaLock = chakra(FaLock);
+const CFMdEmail = chakra(MdEmail);
+const CDFaUserCircle = chakra(FaUserCircle);
```

Comment:
how chakra works for react-icons

Reply:
it works as a wrapper.Gives us the ability to add chakra props .